include/ztd/text/impl/wide_execution_cwchar.hpp

```diff
@@ -466,7 +466,7 @@ namespace ztd { namespace text {
 							             ::std::move(__inlast)),
 							     ::std::move(__result.output), __s, __result.error_code),
 							::ztd::span<code_unit>(::std::addressof(__units[0]), __units_count),
-							__intermediate_handler.__M_code_points_progress());
+							::ztd::span<code_point, 0>());

```

Comment:
(This change, here.)

Reply:
I'm a dingus, that's a dumb fix, it's another ambiguous base problem

Reply:
I changed this to make it compile in the OSX build. Not necessary on Linux, and this change might be what's causing the OSX build to crash...

Anyway, reverting here.

Reply:
What does the error look like on OSX?

Reply:
Idk, it was in the compile logs of the OSX pull. I'll play around with it more in the future or bully one of my students into lending me a Mac to try out on

Reply:
😅 Don't bully your students too much. I'm sure it'll be okay. I can find a mac here to fix it up.